### ISSUE TITLE
Inventories with nested groups can't be loaded

### ISSUE THREAD

**Issue by castroflavio** [ansible/ansible-runner-service#26]
The following line doesn't work for nested groups as the key under group_name is children.
`_host_list.update(list(self.inventory['all']['children'][group_name]['hosts'].keys())) `

**Comment by pcuzner** [comment]
Yeah, the initial use case didn't require nested group support. Is this an issue you'd like to take on?

**Comment by castroflavio** [comment]
Thanks for the reply, now I'm just doing a POC, I may fork your project in the near future.
